package.json

```diff
@@ -67,5 +67,8 @@
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
+  },
+  "dependencies": {
+    "apollo-link-rest": "*"
```

Comment:
We shouldn't add this as a dependecies, we might add this as a `devDependency` for tests